components/SplitScreen/SplitScreen.js

```diff
@@ -77,14 +73,8 @@ function SplitScreen({ inverted, onInvert, reveal, primary, secondary }) {
     config: SPRING_FAST,
     to: {
       transform: `
-        translate3d(
-          0,
-          calc(
-            ${Number(opened) * -50}%
-            - ${(InvertButton.HEIGHT / 2) * Number(opened)}px
-          ),
-          0
-        )`,
+        scale(${opened ? 2 : 1})`,
```

Comment:
This changes the animation style but it feels good to me and solves the hanging shadow.